apps/meteor/tests/data/uploads.helper.ts

```diff
@@ -155,21 +155,21 @@ export async function testFileUploads(
 			})
 			.expect('Content-Type', 'application/json')
 			.expect(200)
-			.expect(function (res: Response) {
+			.expect((res: Response) => {
 				expect(res.body).to.have.property('success', true);
 				expect(res.body).to.have.property('files').and.to.be.an('array');
 			})
 			.end(done);
 	});
 
-	it('should not return thumbnails', async function () {
+	it('should not return thumbnails', async () => {
```

Comment:
are these changes because of our eslint rules? just want to point this out https://mochajs.org/#arrow-functions

I wonder if should have a mix of arrow and non-arrow functions when the `this` context is needed, or if we should stick to one or the other

Reply:
ESLint rules. However, `@typescript-eslint` ensures functions are converted to arrow functions only when there is no references to `this`.
Personally, it feels like there is very little situations where we need to use the Mocha context e.g. `this.retries(n)` was redudant with the current configuration and calls to `this.skip()` are rare and easily replaced with `it.skip`/`describe.skip`.